bots/dev/cli.py

```
@@ -2077,6 +2163,19 @@ def switch(self, bot: Bot, context: CLIContext, args: List[str]) -> str:
 
         return "\n".join(output)
 
+    def clear(self, _bot: Bot, _context: CLIContext, _args: List[str]) -> str:
+        """Clear the terminal screen."""
+        import platform
+        import subprocess
+
+        # Use appropriate clear command for the platform
+        if platform.system() == "Windows":
+            subprocess.run(["cls"], shell=False, check=False)
```

Ruff (0.14.13)

Starting a process with a partial executable path

(S607)